optimade/grammar/v0.10.0.lark

```diff
@@ -48,24 +59,24 @@ number: SIGNED_INT | SIGNED_FLOAT
 // Tokens
 
 // Boolean relations
-_AND: "AND"
-_OR: "OR"
-NOT: "NOT"
+_AND:     "AND"
```

Comment:
I am wondering why it is needed to add the `_` here and for `OR`? Not that it is wrong - I'm just wondering why :)

Reply:
It is a special [feature of the Lark](https://lark-parser.readthedocs.io/en/latest/tree_construction/#shaping-the-tree).  It suppresses it so it won't be the part of the Lark Tree and simplifies the implementation of the `Transformer`.

Reply:
Very cool! Didn't know that feature.